examples/09_cssmodules/src/index.html

```diff
@@ -4,7 +4,7 @@
     <meta charset="utf-8" />
     <title>Waku example</title>
     <meta name="viewport" content="width=device-width, initial-scale=1" />
-    <style>
+    <!-- <style>
```

Comment:
Weirdly enough, having styles in this html would give me a weird error from babel. 

```
Cannot render RSC SyntaxError: /home/aslemammad/oss/waku/examples/09_cssmodules/src/components/Counter.tsx?html-proxy&direct&index=0.css: Support for the experimental syntax 'decorators' isn't currently enabled (2:7):

  1 |
> 2 |       @keyframes spinner {
    |       ^
  3 |         to {
  4 |           transform: rotate(360deg);
  5 |         }

Add @babel/plugin-proposal-decorators (https://github.com/babel/babel/tree/main/packages/babel-plugin-proposal-decorators) to the 'plugins' section of your Babel config to enable transformation.
If you want to leave it as-is, add @babel/plugin-syntax-decorators (https://github.com/babel/babel/tree/main/packages/babel-plugin-syntax-decorators) to the 'plugins' section to enable parsing.
```